packages/cli/src/lib/errors/snapshotErrors.ts

```diff
@@ -1,13 +1,50 @@
 import {blueBright} from 'chalk';
+import {cli} from 'cli-ux';
 import dedent from 'ts-dedent';
 import {Configuration} from '../config/config';
 import {Snapshot} from '../snapshot/snapshot';
 import {SnapshotUrlBuilder} from '../snapshot/snapshotUrlBuilder';
 
-export class SnapshotOperationTimeoutError extends Error {
+// type severityLevels = 'info' | 'warn' | 'error';
+
+enum severityLevels {
+  'info' = 'info',
+  'warn' = 'warn',
+  'error' = 'error',
+}
```

Comment:
https://github.com/microsoft/TypeScript/wiki/Coding-guidelines
Please follow the guideline when possible ;)

All does not makes sense, but for enum here it kinda does (tho flavor, y'know), especially because we followed them so far (for enums)

Reply:
In this case, the enum values need to match the log levels available under the `cli` object. That's because we are doing
I think `type severityLevels = 'info' | 'warn' 'error'` would be more appropriate than enum here.

What do you think?

Reply:
But... You can just do:
```ts
enum SeverityLevel {
  Info = 'info',
  Warm = 'warn',
  Error = 'error',
}
```
No?..

Reply:
Oh I see. I thought you wanted the values to be in pascal case... my bad